sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -96,14 +110,24 @@ def __init__(
         average_precision=None,
         estimator_name=None,
         pos_label=None,
+        prevalence_pos_label=None,
     ):
         self.estimator_name = estimator_name
         self.precision = precision
         self.recall = recall
         self.average_precision = average_precision
         self.pos_label = pos_label
+        self.prevalence_pos_label = prevalence_pos_label
 
-    def plot(self, ax=None, *, name=None, **kwargs):
+    def plot(
+        self,
+        ax=None,
+        *,
+        name=None,
+        plot_chance_level=False,
+        chance_level_kw=None,
```

Comment:
I'm not a huge fan of this parameter name. Unfortunately I also don't have an alternative that I can suggest :( The best I could come up with is `chance_level_plot_args`?

I think my main point is that I read `chance_level_kw` as "chance level keyword". And then think "keyword? For what? Is there a dictionary involved somewhere?". I had to go and read the docstring properly to understand that these are keyword arguments that get passed to `plot()`.

Can we get inspiration for a naming scheme/system/pattern from other functions in scikit-learn that do something similar?

Reply:
> Can we get inspiration for a naming scheme/system/pattern from other functions in scikit-learn that do something similar?

The pattern used here is the one used on all other displays, with the oldest precedence for the `PartialDependenceDisplay` (e.g. https://scikit-learn.org/stable/modules/generated/sklearn.inspection.PartialDependenceDisplay.html#sklearn.inspection.PartialDependenceDisplay.from_estimator).

>  I had to go and read the docstring properly to understand that these are keyword arguments that get passed to plot()

Thus, for all displays, the `*_kw` parameter is used to control some matplotlib rendering. At least this is the most consistent thing that we have right now.

Reply:
I am not against revisiting the naming of the displays. I think that we have currently an inconsistency where sometimes without a generic `**kwargs` that is passed (e.g. for the current plot) that has been replaced by an argument taking a `dict` and making it explicit.

If we have something that makes the naming super straightforward, then it might be worth some deprecation.